src/components/form/RangeSliderField.tsx

```diff
@@ -0,0 +1,46 @@
+import { Path, UseFormRegister } from 'react-hook-form';
+
+import './RangeSliderField.scss';
+
+type RangeSliderFieldProps<TFormValues> = {
+  label: string;
+  name: Path<TFormValues>;
+  register: UseFormRegister<TFormValues>;
+  min: number;
+  max: number;
+  disabled?: boolean;
+};
+
+export const RangeSliderField = <TFormValues extends Record<string, unknown>>({
+  label,
+  name,
+  register,
+  min,
+  max,
+  disabled = false,
+}: RangeSliderFieldProps<TFormValues>) => {
+  return (
+    <>
```

Comment:
Do you prefer using `<>` over `<div>` as a component 'wrapper'. I've had confusing experiences before when using `<>` has messed up my stylings.

Reply:
I probably mix and match if I'm honest. I guess it is better to fully understand what `<></>` and <Fragment></Fragment>` mean. So you'll already know that every component should start with a single root element, whether that be a `ul`, `div`, `span` etc, yes you can code your component to return an array of ReactNodes, but I've always found that to be bad. What `<></>` and it's companion `<Fragment></Fragment> do, the [docs](https://react.dev/reference/react/Fragment) have a good explanation, but basically it is just a way to group elements, however, the big thing to remember is it doesn't wrap the DOM elements, as an example

```html
<form>
  <>
    <label>My Field</label>
    <input type="text" />
  </>
</form>
```

Produces the DOM:

```html
<form>
  <label>My Field</label>
  <input type="text" />
</form>
```
 whereas using a div would produce

```html
<form>
  <div>
    <label>My Field</label>
    <input type="text" />
</div>
</form>
```
Of course, div's come with default styling, so that is why you may see messed up stylings, as you maybe expecting the children to be wrapped in the DOM, but they are not.

This is just me reading between the lines of course, you may already be aware of all this and mean something else. I often use fragments in my code and not had any trouble